libs/hdf-converters/data/owasp-nist-mapping.json

```diff
@@ -0,0 +1,72 @@
+[
```

Comment:
How were these files generated?

Reply:
Using a csv2json converter, e.g https://github.com/julien-f/csv2json

Reply:
If you are going to maintain both of these files in this directory, something should be implemented to ensure these don't get out of sync, similar to how the code is generated for inspecjs.